packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -142,6 +142,59 @@
                     "ja": "警告：あなたは現在、Trezor Suiteの偽バージョンを使用しています。このプログラムを直ちに終了してください。"
                 }
             }
+        },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "22.4.3",
+                        "mobile": "!",
+                        "web": "22.4.3",
+                        "revision": "969161f41dcd889fa1fb0cf5e1d3437fddd8c26d"
+                    },
+                    "devices": [
+                        {
+                            "model": "T",
+                            "firmware": "*",
+                            "bootloader": "2.0.4",
```

Comment:
Once we release 2.0.4 at some point in future, we must not forget removing this. But that is far away.

Reply:
Would that be viable to skip it?